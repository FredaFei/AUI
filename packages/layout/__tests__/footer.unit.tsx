import * as renderer from 'react-test-renderer'
import React from 'react'
// import {shallow} from 'enzyme'

import Footer from '../footer'

describe('测试 Header 以下行为', () => {
  it('Footer 存在', () => {
    const json = renderer.create(<Footer>Footer</Footer>).toJSON()
    expect(json).toMatchSnapshot()
  })
  // it('Footer 设置 自定义类名', () => {
  //   const wrapper = shallow(<Footer className="test">Col-12</Footer>)
  //   expect(wrapper.find('.am-footer').hasClass('test')).toEqual(true)
  // })
  // it('Footer 设置 自定义样式', () => {
  //   const wrapper = shallow(<Footer style={{color: 'red'}}>Col-12</Footer>)
  //   expect(wrapper.render()).toMatchSnapshot()
  // });
})