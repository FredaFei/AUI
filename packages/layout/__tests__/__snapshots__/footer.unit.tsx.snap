// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`测试 Header 以下行为 Footer 存在 1`] = `
<div
  className="am-footer "
>
  Footer
</div>
`;
